understanding/redundant-entry.html

```diff
@@ -56,191 +55,142 @@
                   
-                  <li>When the previously entered information is no longer valid, it can be requested that
-                     the user enter that information again.
+                  <li>以前に入力された情報が有効でなくなった場合、利用者にその情報を再度入力するように要求できる。
                   </li>
                   
                </ul>
             </section>
             <section id="benefits">
-               <h2>Benefits</h2>
+               <h2>利点</h2>
                <ul>
                   
-                  <li>Users with cognitive disabilities experience short-term, working memory difficulty.
-                     Not having to repeatedly remember particular information reduces stress and the likelihood
-                     of mistakes.
+                  <li>認知障害のある使用者は、短期的な作業記憶の難しさを感じている。特定の情報を繰り返し思い出す必要がないため、ストレスが軽減され、間違いが起こりにくくなる。
                   </li>
                   
-                  <li>Users who experience difficulty forming new memories, recalling information, and other
-                     functions related to cognition can complete processes without having to unnecessarily
-                     rely on their memory.
+                  <li>新しい記憶の形成、情報の想起、および認知に関連するその他の機能に難しさを感じている利用者は、必要以上に記憶に頼ることなくプロセスを完了できる。
                   </li>
                   
-                  <li>Users with mobility impairments, for example using switch control or voice input,
-                     benefit from a reduced need for text entry.
+                  <li>例えば、スイッチ操作、音声入力を使用するような、運動障害のある利用者は、テキストの入力の必要性が軽減されるというメリットがある。
                   </li>
                   
                </ul>
             </section>
             <section id="examples">
-               <h2>Examples</h2>
+               <h2>事例</h2>
                <ul>
                   
-                  <li>A form requests the user’s corporate identification number (ID) in the first step
-                     of a process to purchase a new computer. In the 3rd step the user is asked to confirm
-                     that the computer will belong to the user (rather than a colleague), and re-shows
-                     the ID. It allows the user to change the ID, but defaults to the previously entered
-                     one.
+                  <li>新しいコンピューターを購入するプロセスの最初のステップで、フォームが利用者の法人識別番号 (ID) を要求する。3 番目のステップで、利用者はコンピューターが (同僚ではなく) 利用者のものであることを確認するよう求められ、ID が再表示される。利用者は ID を変更できるが、デフォルトでは以前に入力した ID になっている。
                   </li>
                   
-                  <li>A form on an e-commerce website allows the user to confirm that the billing address
-                     and delivery address are the same address.
+                  <li>電子商取引サイトのフォームでは、利用者は請求先住所と配送先住所が同じ住所であることを確認できる。
                   </li>
                   
-                  <li>A search results page pre-fills the search input with the previously entered search
-                     term in the same process.
+                  <li>検索結果ページでは、同じプロセスで以前に入力した検索語が検索入力に事前に入力されている。
```

Comment:
fillとinputを訳し分けたい気持ちで

```suggestion
                  <li>検索結果ページでは、同じプロセスで以前に入力した検索語が検索入力欄に事前に補完されている。
```